### PR TITLE
Preparing BV 4.3

### PR DESCRIPTION
**Pull request by srbarrios**
Depends on https://gitlab.suse.de/galaxy/infrastructure/-/merge_requests/461

### CODE REVIEW

File: terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf [hunk 1]
```diff
@@ -0,0 +1,1387 @@
...
+  required_providers {
+    libvirt = {
+      source = "dmacvicar/libvirt"
+      version = "0.6.3"
+    }
+  }
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://caladan.mgr.prv.suse.net/system"
```
Comment (ktsamis) [inline terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf:88]: I think we need to find new servers for this BV deployment

**Review by ktsamis — APPROVED** [review]
Except the hypervisors that we can change later, looks great :)

File: terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf (hunk 1 above)
Comment (ktsamis) [reply terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf:88]: Eric will create the new servers and change these lines and then we merge

Comment (Bischoff) [reply terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf:88]: After discussion, merging first, and adding the servers after

**Pull request merged**